neural_lifetimes/data/datasets/sequence_sampling_dataset.py

```diff
@@ -352,7 +352,7 @@ def torchify(x: Dict[str, np.ndarray], device: torch.device) -> Dict[str, Union[
     for k, v in x.items():
         if is_numeric_dtype(v.dtype):
             nice_type = normalize_types(v.dtype)
-            out[k] = torch.tensor(v.astype(nice_type), device=device)
+            out[k] = torch.tensor(v.astype(nice_type))
         else:
```

Comment:
So where do we specify if we are using the CPU or GPU?

Reply:
Lightning automatically selects the accelerator given what is available. if you want it to force to run on a certain accelerator, you just use `pl.Trainer(accelerator='tpu')`